recipe/meta.yaml

```
@@ -22,11 +23,11 @@ requirements:
   run:
     - python
     - cloudpickle
-    - ipykernel >=6.16.1,<7.0.0
-    - ipython >=7.31.1,<9.0.0
+    - ipykernel >=6.23.2,<7.0.0
+    - ipython >=7.31.1,<9.0.0,!=8.8.0,!=8.9.0,!=8.10.0,!=8.11.0,!=8.12.0,!=8.12.1
     - jupyter_client >=7.4.9,<9.0.0
     - pyzmq >=22.1.0
-    - wurlitzer >=1.0.3  # [not win]
+    - wurlitzer >=1.0.3  # [unix]
```

```suggestion
    - wurlitzer >=1.0.3  # [not win]
```

`'wurlitzer>=1.0.3;platform_system!="Windows"',`

https://github.com/spyder-ide/spyder-kernels/blob/0c4ee29ef195d24ac66455fa7e253a4cd1ab1a8c/setup.py#L50

it could be for `osx` as well (posix: https://github.com/spyder-ide/spyder-kernels/blob/v2.4.4/requirements/posix.txt)

---

@lorepirri `osx` is considered `unix`. But yes, to match what upstream does, `not win` is preferable.

---

Ah right, thanks @JeanChristopheMorinPerso

---

This has been updated